### ISSUE TITLE
error message improvement

### ISSUE THREAD

**Issue by bertvannuffelen** [oeg-upm/fair_ontologies#106]
On the first check:

```
 PURL1: Persistent URL

Description
    This check verifies if the ontology has a persistent URL (w3id, purl, DOI, or a W3C URL) 
Explanation
    Ontology URI is not using a persistent id. We checked w3id, purl, DOI and W3C 
```

The explanation is making a strong statement. 
Better: the ontology URI is not part of a known register of URIs. We checked w3id, purl, DOI and W3C 

And by the way the persistency cannot be determined only by the domain. Both purl and DOI and w3id are redirection services and therefore do not care the persistency. That is the real url owner.

**Comment by dgarijo** [comment]
Thanks for your suggestions!
I agree in that we can improve the wording of the check.

However, purl and w3id do care about the persistence of the URL. It's the
only thing that they ensure (purl stands for permanent urls, if I am not
wrong). They do not check that the URI being redirected to is persistent,
or that it resolves (is there something truly persistent on the web?). But
the URL they provide does not change, hence we consider it persistent, and
a good practice.

A separate check addresses whether the URI resolves in fact to a semantic
artefact.

**Comment by bertvannuffelen** [comment]
I agree that these namespaces have the intention and objective of persistence as basis. 

My annoyance is that this is not necessary the ambition of those who publishes on those domains. 
It occurs too much that the development behind the URI is stalled and restarted on a different place with different persistent URI's.
And that breaks the whole idea of persistency. 

I know no test can proof that this is the case, and that this implemented test (with the additional tests you refer to) are a necessary check to drive towards persistency.  

Therefore as an additional suggestion to promote the efforts of organisations that put long term efforts to maintain persistent URIs I would suggest you also collect these namespaces and include that in your validations. Within the EU there are many public administrations putting a lot of effort into it. By including these in the list you create also an acknowledgement of the effort spend by these organisations.
To name a few: 
   - Publications Office of EU: http://publications.europa.eu/resource
   - EU wide: https://data.europa.eu/URI.html
   - Regional efforts: https://data.vlaanderen.be

**Comment by dgarijo** [comment]
Thanks, I think you make a fair point. We can expand the list pointing to these efforts as well. There is another issue to include governmental URLs from a data service in Australia. I am going to have a look at these resources and see how to incorporate them
